job_executor/worker/steps/dataset_validator.py

```diff
@@ -38,6 +39,11 @@ def run_for_dataset(dataset_name: str) -> Tuple[str, str]:
             'Failed to validate dataset. '
             'Resolve errors with the microdata-validator before uploading.'
         )
+    local_storage.archive_input_files(
```

Comment:
Consider calling this directly from the build_dataset_worker and build_metadata_worker pipelines, after the validator 👍🏻

Reply:
Done